notebook/static/notebook/js/actions.js

```diff
@@ -131,7 +137,7 @@ define(function(require){
                 env.notebook.execute_cells_below();
             }
         },
-        'go-to-command-mode': {
+        'enter-command-mode': {
```

Comment:
`switch` ? Many time I say "Enter command mode" to French speaker, they press Enter. I would like to avoid this confusion.